src/typegraph/deno/src/io.ts

```diff
@@ -7,6 +7,38 @@ import process from "node:process";
  * see: module level documentation `meta-cli/src/deploy/actors/task.rs`
  */
 
+const JSONRPC_VERSION = "2.0";
+
+function writeRpcMessage(message: string) {
+  // split into 32-KiB chunks
+  const chunkSize = 32758; // 32 KiB - 10 bytes for "jsonrpc^: " or "jsonrpc$: "
+  for (let i = 0; i < message.length; i += chunkSize) {
+    const chunk = message.slice(i, i + chunkSize);
```

Comment:
Shouldn't we encode the message first? Because we can definitely get variable length characters.

Reply:
Actually nevermind, I forgot that you took a margin and that JavaScript correctly handles UTF-8.